components/script/script_module.rs

```diff
@@ -280,7 +280,7 @@ impl ModuleTree {
     }
 
     pub fn remove_incomplete_fetch_url(&self, dependency: ServoUrl) {
-        self.incomplete_fetch_urls.borrow_mut().remove(&dependency);
+        self.incomplete_fetch_urls.borrow_mut().swap_remove(&dependency);
```

Comment:
from doc:
/// This is equivalent to [`.swap_remove(value)`][Self::swap_remove], replacing this
/// value's position with the last element, and it is deprecated in favor of calling that
/// explicitly.

Reply:
We're not relying on the order of `incomplete_fetch_urls` though, as we're using `IndexSet`, I wonder maybe it's better to move to `shift_remove` (in case that folks may expect the order is preserved)? 

```suggestion
        self.incomplete_fetch_urls.borrow_mut().shift_remove(&dependency);
```

Reply:
Thank you for the update! As the first wpt try is triggered by previous commit, let me trigger them again.